### ISSUE TITLE
Auto-correct of Layout/RescueEnsureAlignment ineffective

### ISSUE THREAD

**Issue by Drenmi** [rubocop-hq/rubocop#6258]
The auto-correct of `Layout/RescueEnsureAlignment` is ineffective, as it rewrites the code the exact same way.

--------

## Expected behavior

Auto-correct to:

```ruby
class Foo
  def bar
    @bar ||= begin
               expensive_method
             rescue StandardError
               fall_back
             end
  end
end
```

## Actual behavior

The auto-correct is a no-op.

## Steps to reproduce the problem

Auto-correct the following with `Layout/RescueEnsureAlignment`:

```ruby
class Foo
  def bar
    @bar ||= begin
      expensive_method
    rescue StandardError
      fall_back
    end
  end
end
```

## RuboCop version

```
0.59.0 (using Parser 2.5.1.2, running on ruby 2.5.1 x86_64-darwin16)
```

**Comment by andrew-aladev** [comment]
`RescueEnsureAlignment` aims to align `rescue` keyword with `begin`. I don't think it could be implemented within `RescueEnsureAlignment`. It a job for another cop.

**Comment by MikeMcQuaid** [comment]
Homebrew would still find this useful so just commenting so stale bot knows someone cares.
